### PR TITLE
Feature/short info migrate to hooks and typescript

### PR DESCRIPTION
**Pull request by hacheG**

### CODE REVIEW

File: src/components/ShortInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+// import PropTypes from 'prop-types';
```
Comment (cazapatamar) [inline src/components/ShortInfo.tsx:2]: Delete comments

File: src/components/ShortInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+//   tooltip: PropTypes.string,
+//   collapseButton: PropTypes.bool,
+// };
+
+// ShortInfo.defaultProps = {
+//   className: 'hidden',
+//   description: '',
+//   tooltip: '',
+//   collapseButton: true,
+// };
```
Comment (cazapatamar) [inline src/components/ShortInfo.tsx:105]: Delete comments

Comment (cazapatamar) [inline src/components/ShortInfo.tsx:105]: Include the default values

**Review by cazapatamar — CHANGES_REQUESTED** [review]

File: src/components/ShortInfo.tsx (hunk 1 above)
Comment (hacheG) [reply src/components/ShortInfo.tsx]: ok, I deleted the comments, sorry about that

File: src/components/ShortInfo.tsx (hunk 2 above)
Comment (hacheG) [reply src/components/ShortInfo.tsx]: ok, I deleted the comments, sorry about that

Comment (hacheG) [reply src/components/ShortInfo.tsx]: I put "hidden" default value in props className

File: src/components/ShortInfo.jsx [hunk 3]
```diff
@@ -1,62 +0,0 @@
...
-  className: PropTypes.string,
-  description: PropTypes.string,
-  tooltip: PropTypes.string,
-  collapseButton: PropTypes.bool,
-};
-
-ShortInfo.defaultProps = {
-  className: 'hidden',
-  description: '',
-  tooltip: '',
```
Comment (cazapatamar) [inline src/components/ShortInfo.jsx:58]: You are still missing the default values for `description` and `tooltip` attribute. Remember that an empty string is different to a `null` or `undefined` value.

**Review by cazapatamar — CHANGES_REQUESTED** [review]

File: src/components/ShortInfo.jsx (hunk 3 above)
Comment (hacheG) [reply src/components/ShortInfo.jsx:58]: done

File: src/components/ShortInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import isUndefinedOrNull from "utils/validations";
+
+interface ShortInfoTypes {
+  className: string;
+  description: string;
+  tooltip?: string;
+  collapseButton: boolean;
```
Comment (cazapatamar) [inline src/components/ShortInfo.tsx:8]: Why do you only set as non-mandatory the attribute `tootip`?. Previously, no attribute was marked as mandatory.

**Review by cazapatamar — CHANGES_REQUESTED** [review]

File: src/components/ShortInfo.tsx (hunk 4 above)
Comment (hacheG) [reply src/components/ShortInfo.tsx:8]: tooltip var works in home but in ecosystems, for example, it does not have a value

Comment (cazapatamar) [reply src/components/ShortInfo.tsx:8]: Previously, no prop was marked as mandatory. Set the same logic please.

**Review by cazapatamar — APPROVED** [review]

**Comment by cazapatamar** [comment]
Please dont merge until @ManuelStardust approve

File: src/components/ShortInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  const handleClick = () => {
+    setRotate_button(!rotate_button);
+    setHide_text(!hide_text);
+  };
+
+  return (
+    <div>
+      <div
+        className={`${className}-${hide_text}`}
+        // eslint-disable-next-line react/no-danger
```
Comment (ManuelStardust) [inline src/components/ShortInfo.tsx:29]: please delete this comment

**Review by ManuelStardust — CHANGES_REQUESTED** [review]

File: src/components/ShortInfo.tsx (hunk 5 above)
Comment (hacheG) [reply src/components/ShortInfo.tsx]: Done!

**Review by ManuelStardust — APPROVED** [review]

**Review by cazapatamar — APPROVED** [review]

**Pull request merged**